ext/opentelemetry-ext-django/tests/test_middleware.py

```diff
@@ -111,3 +114,25 @@ def test_error(self):
             span.attributes["http.url"], "http://testserver/error/"
         )
         self.assertEqual(span.attributes["http.scheme"], "http")
+
+    @patch(
+        "opentelemetry.ext.django.middleware._DjangoMiddleware._excluded_urls",
+        ExcludeList(["http://testserver/excluded_arg/123", "excluded_noarg"]),
```

Comment:
Should add a test using some more regex functionality ^ and $ will probably be used often.

Reply:
added specific regex tests in the api instead of the ext